src/ports/tilix/index.ts

```diff
@@ -0,0 +1,27 @@
+import { AuraAPI } from 'core'
+import { resolve } from 'path'
+
+export async function TilixPort(Aura: AuraAPI) {
+  const { createPort, createReadme, colorSchemes, constants } = Aura
+  const templateFolder = resolve(__dirname, 'templates')
+  const { info } = constants
+
+  const portName = 'Tilix'
+  const version = '1.0.0'
+
```

Comment:
Add this line and pass the `previewURL` const to the **createReadme** function replacements:
```js
const previewURL = `https://github.com/${info.author.username}/assets/blob/master/images/${info.slug}/aura-tilix-preview.png?raw=true`
```